<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1980</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Monaco</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>EP_lab4</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(224, 212, 255);</string>
  </property>
  <widget class="QPushButton" name="calc_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>320</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="title">
    <string>Параметры</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label_req_cnt">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Время моделирования</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_edit_time">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>800</y>
     <width>871</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Точка для проверки</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>851</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="1" column="4">
      <widget class="QLineEdit" name="line_edit_x4_full">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="line_edit_x2_full">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="line_edit_x3_full">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_a_13">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="line_edit_x1_full">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_a_14">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="label_a_26">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_a_7">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLineEdit" name="line_edit_x6_full">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_a_12">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_a_25">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="line_edit_x5_full">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_a_15">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_a_9">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLineEdit" name="line_edit_x7_full">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QLineEdit" name="line_edit_x8_full">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="check_full_button">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>850</y>
     <width>311</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="text">
    <string>Проверить</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>1821</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTextEdit" name="line_edit_nonlin_regr">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(230, 212, 255);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_b_9">
      <property name="font">
       <font>
        <family>Monaco</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Нелинейное уравнение </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_a_8">
      <property name="font">
       <font>
        <family>Monaco</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Постоянная S</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_b_8">
      <property name="font">
       <font>
        <family>Monaco</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Длина звездного плеча</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line_edit_s">
      <property name="font">
       <font>
        <family>Monaco</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(230, 212, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="line_edit_s_length">
      <property name="font">
       <font>
        <family>Monaco</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(230, 212, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="title">
    <string>Первый генератор</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>317</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QLabel" name="label_a_2">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">дисперсия</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>max</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="line_edit_max_gen_var">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_edit_min_gen_int">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1.5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_b">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">интенсивность  </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_edit_min_gen_var">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="line_edit_max_gen_int">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>40</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="title">
    <string>Обработчик заявок 1-го типа</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="0">
      <widget class="QLabel" name="label_a_3">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">дисперсия </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_edit_min_pm_var_1">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="line_edit_max_pm_var_1">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_a_4">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">интенсивность  </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_edit_min_pm_int_1">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="line_edit_max_pm_int_1">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="title">
    <string>Второй генератор</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>351</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0">
      <widget class="QLabel" name="label_a_6">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">интенсивность  </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="line_edit_max_gen_var_2">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="line_edit_max_gen_int_2">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_edit_min_gen_int_2">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_edit_min_gen_var_2">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_a_5">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">дисперсия</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>40</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monaco</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 212, 255);</string>
   </property>
   <property name="title">
    <string>Обработчик заявок 2-го типа</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="1" column="2">
      <widget class="QLineEdit" name="line_edit_max_pm_int_2">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_edit_min_pm_int_2">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_edit_min_pm_var_2">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_a_23">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">дисперсия</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_a_24">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">интенсивность  </string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="line_edit_max_pm_var_2">
       <property name="font">
        <font>
         <family>Monaco</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
